src/modules/esl-media/core/esl-media.ts

```diff
@@ -81,6 +82,8 @@ export class ESLMedia extends ESLBaseElement {
   @attr({parser: parseLazyAttr, defaultValue: 'none'}) public lazy: ESLMediaLazyMode;
   /** Autoplay resource marker */
   @boolAttr() public autoplay: boolean;
+  /** Will prevent autoplay if media was paused manually */
+  @boolAttr() public noAutoplayOnInterrupt: boolean;
```

Comment:
Actually, my only big concern is the name and default for that variable...
Regarding default I propose invert of the option...
Regarding naming
   I'd consider special string value for autoplay (e.g. ``autoplay="always"` and doc note about its behaviour)
      
Another option is `autoplay-on-interrupt` but it is still long

@exadel-inc/esl-all-contributors feel free to jump with suggestions and comments here